controllers/machinedeletionremediation_controller.go

```diff
@@ -546,14 +569,14 @@ func (r *MachineDeletionRemediationReconciler) getMachineSetNodes(machineSetName
 			continue
 		}
 
-		if thisMachineSetName, err := getMachineOwnerReferenceName(&machine); err != nil {
+		if thisOwnerName, _, err := getMachineOwnerReference(&machine); err != nil {
```

Comment:
NIT: Can we use different var name, without the word 'this', as it reminds the `this` keyword from other languages and it seems a little bit misleading IMHO. An alternative would be to use `expectedOwnerName` and `fetchedOwnerName`/`currentOwnerName`/`ownerName` or anything equivalent

Reply:
`curOwnerName`?